package.json

```diff
@@ -4,14 +4,21 @@
   "description": "E4P JumboCode Project",
   "main": "server.js",
   "scripts": {
+    "lint-all": "npm run lint-client && npm run slint-client && npm run hlint-client",
     "test": "mocha --exit --recursive tests",
     "start": "node server.js",
     "slint": "stylelint",
-    "slint-client": "stylelint ./public/**/*.css"
+    "slint-client": "stylelint ./public/**/*.css",
```

Comment:
Are all of these necessary? It seems like many are unused by lint-all (like lint, hlint, lint-server, lint-tests). Should those be included to the lint-all command?